recipes/procps-ng/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+./autogen.sh
+./configure --prefix=$PREFIX --enable-watch8bit
+
+make
```

Comment:
Is there anything preventing this from being a parallel build?
```suggestion
make -j${CPU_COUNT}
```

Reply:
That should work fine, I presume.